tests/codeception/backend/functional/LoginCept.php

```diff
@@ -17,7 +17,6 @@
 $I->see('Password cannot be blank.', '.help-block');
 
 $I->amGoingTo('try to login with wrong credentials');
-$I->expectTo('see validations errors');
 $loginPage->login('admin', 'wrong');
 $I->expectTo('see validations errors');
```

Comment:
Here `expectTo()` seems to be unnecessary duplicated indeed.
However it should be better to remove the second entry instead of first one, so scebario description methods are tight to each other.